File: Clients/Android/app/src/main/java/Models/LearningResources/GrammarTask.java
package Models.LearningResources;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

/**
 * Created by MARKAN on 10.05.2017.
 */
public class GrammarTask implements Serializable {
    @SerializedName("Text")
    public String text;
    @SerializedName("Translate")
    public String translate;
    @SerializedName("Description")
    public String description;
    @SerializedName("Answers")
    public List<GrammarAnswer> answers = null;

    public GrammarTask(String text, String translate, String description, List<GrammarAnswer> list) {
        this.answers = list;
        this.description = description;
        this.text = text;
        this.translate = translate;
    }
}

File: Server/EasyUkr.WebApi/Models/RestModels/GrammarTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace EasyUkr.WebApi.Models.RestModels
{
    public class GrammarTask
    {
        public string Text { get; set; }
        public string Translate { get; set; }
        public string Description { get; set; }
        public ICollection<GrammarAnswer> Answers { get; set; }
    }

    public class GrammarAnswer
    {
        public string Text { get; set; }
        public bool IsCorrect { get; set; }
    }
}